resources/js/components/pages/Settings.vue

```diff
@@ -55,11 +55,21 @@
                         description="Enable advanced mode."
                         readmore="Advanced mode"
                     />
-                    <Btn v-if="appStore.isMultiTenant" class="mr-auto" @click="logout">Logout</Btn>
-                    <Btn v-else class="mr-auto" error @click="resetSettings"> Reset Settings </Btn>
+                    <div class="flex space-x-4">
+                        <Btn v-if="!appStore.isMultiTenant" @click="confirmModal = true"> Delete account </Btn>
```

Comment:
I deployed the latest changes on staging but this button doesn't appear. I think it's because of this `!appStore.isMultiTenant`